java/server/src/org/openqa/grid/common/defaults/DefaultNode.json

```diff
@@ -30,11 +30,20 @@
           "browserName": "internet explorer",
           "maxInstances": 1,
           "seleniumProtocol": "WebDriver"
+        },       
+        {
+          "browserName": "MicrosoftEdge",
+          "platform": "WINDOWS",
+          "maxInstances": 5,
+          "seleniumProtocol": "WebDriver"
         }
       ],
   "configuration":
   {
     "proxy": "org.openqa.grid.selenium.proxy.DefaultRemoteProxy",
+    "Dwebdriver.ie.driver=C://Selenium/Node/IEDriverServer32.exe": "",
```

Comment:
we should not be coding into the default config these paths. Normally it's recommended to just put these executables on the system PATH. Also, this assumes windows, which is not something we can assume at that point (even though it configures 'ie')